scripts/get-workspace-map.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# get-workspace-map.sh
+#   Read Namespace resources and generate a JSON object mapping from namespaces
+#   to workspace based on the value of label toolchain.dev.openshift.com/space.
+#   This script assumes `oc` is preconfigured with all the required clusters
+#   and could be found in $PATH.
+#
+#   if the CONTEXTS environment variable is set, the script will query
+#   locally-defined contexts with those names. Otherwise, the script will query
+#   contexts associated with a hard-coded list of staging clusters.
+#
+set -o pipefail -o errexit -o nounset
+
+# Space-separated locally-defined context names to be queried
+CONTEXTS=(${CONTEXTS:-""})
```

Comment:
Lets document this like we document the parameters for other scripts

Reply:
done

Reply:
I now changed the way the contexts env var is expanded and evaluated. It should be safer now, and should now also attend to the case in which somehow the first provided context is an empty string.

https://github.com/redhat-appstudio/segment-bridge/compare/a73b70c6517b28af9283f3a6634cfa80e37d4f0d..07d6a565fe32595cfde8ad45607cb51e4c617ea3

Reply:
Also changed the way the array of contexts is being split from the jq command, so ShellCheck don't complain.

https://github.com/redhat-appstudio/segment-bridge/compare/07d6a565fe32595cfde8ad45607cb51e4c617ea3..533ff4f7b86c5e0ecdd2c5234fe756940d9d455d